src/hotspot/share/runtime/arguments.cpp

```diff
@@ -3645,6 +3645,11 @@ jint Arguments::parse(const JavaVMInitArgs* initial_cmd_args) {
 
   apply_debugger_ergo();
 
+  // The VMThread needs to stop now and then to execute these debug options.
+  if (HandshakeALot && SafepointALot && FLAG_IS_DEFAULT(GuaranteedSafepointInterval)) {
```

Comment:
Should this be (HandshakeALot || SafepointALot) instead of HandshakeALot && SafepointALot?

Reply:
Yes it should be.